data/text/common.asm

```diff
@@ -1634,6 +1634,15 @@ _NotYetText::
 	text "Not yet…"
 	done
 
+SECTION "_FoundAnEggAgainText", ROMX
+_FoundAnEggAgainText::
+	text "Hello <PLAYER>!"
+	line "Your #mon had"
+
+	para "another egg."
+	line "You want it?"
```

Comment:
text "Ah, it's you!"

	para "Your #mon had"
	line "another Egg!"

	para "You want it?"